server/meeting.go

```diff
@@ -61,6 +65,53 @@ func (p *Plugin) SaveMeeting(meeting *Meeting) error {
 	return nil
 }
 
+func (p *Plugin) calculateQueueItemNumberAndUpdateOldItems(meeting *Meeting, args *model.CommandArgs, hashtag string) (int, error) {
+	c, appErr := p.API.GetChannel(args.ChannelId)
+	if appErr != nil {
+		return 0, appErr
+	}
+	terms := fmt.Sprintf("in:%s %s", c.Name, hashtag)
+	searchResults, appErr := p.API.SearchPostsInTeamForUser(args.TeamId, args.UserId, model.SearchParameter{Terms: &terms})
+	if appErr != nil {
+		return 0, errors.Wrap(appErr, "Error calculating list number")
+	}
+
+	counter := 1
+
+	var sortedPosts []*model.Post
+	// TODO we won't need to do this once we fix https://github.com/mattermost/mattermost-server/issues/11006
+	for _, post := range searchResults.PostList.Posts {
+		sortedPosts = append(sortedPosts, post)
+	}
+
+	sort.Slice(sortedPosts, func(i, j int) bool {
+		return sortedPosts[i].CreateAt < sortedPosts[j].CreateAt
+	})
+
+	for _, post := range sortedPosts {
+		_, parsedMessage, err := parseMeetingPost(meeting, post)
+		if err != nil {
+			p.API.LogDebug(err.Error())
+			return 0, errors.New(err.Error())
+		}
+
+		_, updateErr := p.API.UpdatePost(&model.Post{
+			Id:        post.Id,
+			UserId:    post.UserId,
+			ChannelId: post.ChannelId,
+			RootId:    post.RootId,
+			Message:   fmt.Sprintf("#### %v %v) %v", hashtag, counter, parsedMessage.textMessage),
+		})
+
+		if updateErr != nil {
+			return 0, errors.Wrap(updateErr, "Error updating post")
+		}
+		counter++
```

Comment:
Don't feel obligated to implement this suggestion, but the newlines are meant to separate different operations. Since this `updateErr` is logically coupled with the "block" above, we don't need the newline between them. The block above actually not even a block, it's a function call with a struct. If it was an actual loop/if statement then it would make since to insert a newline after it.

Suggestion: We can remove the newline _above_ this if statement. But we should always have a newline _after_ if statements and loops. This is because after the block is finished, we are done both checking the value and processing its conditional logic. We then move on to something unrelated, thus the newline helps signal this intention.

```suggestion
		})
		if updateErr != nil {
			return 0, errors.Wrap(updateErr, "Error updating post")
		}

		counter++
```

Reply:
@sanjaydemansol  Thank you so much for highlighting that; I understood. Newlines are finally demystified for me.😊